types/string.prototype.repeat/index.d.ts

```diff
@@ -1,3 +1,4 @@
 declare module 'string.prototype.repeat' {
-  export = typeof Function.call.bind(String.prototype.repeat);
+  function repeat(text: string, count: number): string;
```

Comment:
this means that it won't always inherit the builtin type for repeat. is this necessary?

Reply:
The previous version of `types/string.prototype.repeat/index.d.ts` doesn't work. 

```ts
declare module 'string.prototype.repeat' {
  export = typeof Function.call.bind(String.prototype.repeat);
           // ^: The expression of an export assignment must be an identifier or qualified name in an ambient context.ts(2714)
}
```

This error didn't show up because `tsconfig.json` didn't include `types/` at all, thus `string.prototype.repeat` was been treated as `any` by TypeScript.

I'm not sure if there are better way to write this `index.d.ts`, but it definitely need some changes.

Reply:
ah, right. how about:
```suggestion
  const repeat: typeof Function.call.bind(String.prototype.repeat);
```
?

Reply:
It doesn't work :(

```
declare module 'string.prototype.repeat' {
  const repeat: typeof Function.call.bind(String.prototype.repeat);
                                      // ^: ',' expected. ts(1005)
  export = repeat;
}
```

Reply:
i can fix this later, probably by making a DT package.